src/webrtc/callFeed.ts

```diff
@@ -295,6 +298,16 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
             this.analyser = null;
             releaseContext();
         }
+        this._disposed = true;
+        this.emit(CallFeedEvent.Disposed);
+    }
+
+    public get disposed(): boolean {
+        return this._disposed;
+    }
+
+    public set disposed(value: boolean) {
```

Comment:
No need for this to be public though I think?

Reply:
Argh, my mind a few seconds ago: "this should definitely be private" and then I just forgot